### DISCUSSION TITLE
Importing a backup from Simple Tab Groups

### DISCUSSION THREAD

**Discussion by quazar-omega** [mbnuqw/sidebery#1456] [Ideas]
I'd like to move away from Simple Tab Groups to Sidebery on a device of mine, but I have many groups and tabs in them, so I was wondering if there was an easy way to import an STG backup into Sidebery and have all groups be converted into panels

**Comment by tech189** [comment]
I was looking to do the same so I wrote a Python program to convert a Simple Tab Groups backup to a Sidebery snapshot! It uses any old Sidebery snapshot (doesn't matter what's in it - it gets deleted). Let me know if it works for you too.

Instructions and code here: https://gist.github.com/tech189/252d9e9ec8942ada0a52c041e24a0e12

**Reply by quazar-omega** [reply]
Oh, I somehow missed your comment, this is cool! I'll try it and let you know
